### PR TITLE
ARM Templates for Delphix Data Compliance v1.1

### PR DESCRIPTION
**Pull request by ankit-patil-hubs**
### Context
This PR adds additional features to the two ARM templates for data compliance using the Delphix Continuous Compliance Platform.

### Template for Sensitive Data Discovery with Delphix
This template can be used to configure a Delphix Continuous Compliance Engine with what it requires to run sensitive data discovery, also known as Profiling. Profiling is the process that identifies columns that may contain sensitive data and assign appropriate algorithms to mask this data.

Additionally, this template includes automation of PreSetup steps like the creation of an environment, application, and data location to enable the profiling.

### Template for Data Masking with Delphix
This template extracts data from a source data store executes Delphix Masking Job(s) to mask sensitive data in the extract using appropriate algorithms, and finally, loads the compliant data into the target datastore.

### Testing
These templates have been validated with Azure SQL DB as the source and Synapse as the target.

### CODE REVIEW

**Comment by ankit-patil-hubs** [comment]
@microsoft-github-policy-service agree [company="Delphix"]

**Comment by ankit-patil-hubs** [comment]
@microsoft-github-policy-service agree company="Delphix"

**Comment by nabhishek** [comment]
@fhljys Looks good for merging!

File: community templates/Data Masking with Delphix/manifest.json [hunk 1]
```diff
@@ -1,45 +1,43 @@
...
...offset=\"0.64\" stop-color=\"#0074CD\"/><stop offset=\"0.82\" stop-color=\"#006ABB\"/><stop offset=\"1\" stop-color=\"#005BA1\"/></linearGradient><linearGradient x1=\"23.7578605\" x2=\"135.72050000000002\" y1=\"88.78556086375\" y2=\"88.78556086375\" id=\"GRAD3338739\" gradientUnits=\"userSpaceOnUse\"><stop offset=\"0\" stop-color=\"#005BA1\"/><stop offset=\"0.07\" stop-color=\"#0060A9\"/><stop offset=\"0.36\" stop-color=\"#0071C8\"/><stop offset=\"0.52\" stop-color=\"#0078D4\"/><stop offset=\"0.64\" stop-color=\"#0074CD\"/><stop offset=\"0.82\" stop-color=\"#006ABB\"/><stop offset=\"1\" stop-color=\"#005BA1\"/></linearGradient><linearGradient x1=\"1080.9910918874998\" x2=\"2440.573739625\" y1=\"77190.47074375\" y2=\"77190.47074375\" id=\"GRAD3338740\" gradientUnits=\"userSpaceOnUse\"><stop offset=\"0\" stop-color=\"#005BA1\"/><stop offset=\"0.07\" stop-color=\"#0060A9\"/><stop offset=\"0.36\" stop-color=\"#0071C8\"/><stop offset=\"0.52\" stop-color=\"#0078D4\"/><stop offset=\"0.64\" stop-color=\"#0074CD\"/><stop offset=\"0.82\" stop-color=\"#006ABB\"/><stop offset=\"1\" stop-color=\"#005BA1\"/></linearGradient><clipPath id=\"CLIP3338741\"><path d=\"M 3.276,3.276 L 17.724,3.276 L 17
```
Comment (zhimadaren) [inline community templates/Data Masking with Delphix/manifest.json:34]: please don't forget to add these two properties

File: community templates/Sensitive Data Discovery with Delphix/manifest.json [hunk 2]
```diff
@@ -1,40 +1,38 @@
...
...999998 1.459,2.6580000000000004 1.5349999999999997,2.74 L 3.671,5.032 L 7.478010000000001,0.15700000000000003 C 7.555010000000001,0.05799999999999983 7.671010000000001,0 7.79701,0 C 7.88701,0 7.972010000000001,0.028999999999999915 8.04401,0.08399999999999963 L 8.95001,0.7830000000000004 C 9.12301,0.9109999999999996 9.16001,1.1600000000000001 9.03001,1.3369999999999997 L 4.161999999999999,7.57 C 3.9730000000000003,7.811999999999999 3.6119999999999997,7.827999999999999 3.4019999999999997,7.603 L 0.1080000000000001,4.0649999999999995 z\" fill=\"#107C10\" stroke=\"none\" transform=\"matrix(1, 0, 0, 1, 1.4849949999999996, 2.1985)\"/></g></g><g transform=\"matrix(1, 0, 0, 1, 0, 36)\"><g transform=\"matrix(1, 0, 0, 1, 0, 6)\"><g transform=\"matrix(1, 0, 0, 1, 0, 0)\" filter=\"url(#SHADOW2519527)\"><filter id=\"SHADOW2519527\" x=\"-100%\" y=\"-100%\" width=\"300%\" height=\"300%\"><feGaussianBlur in=\"SourceAlpha\" result=\"blur\" LA=\"1.5\"/><feFlood in=\"blur\" result=\"flood\" flood-color=\"#0078dA\"/><feComposite in=\"flood\" in2=\"blur\" operator=\"in\" result=\"comp\"/><feOffset in=\"comp\" result=\"offsetBlur\" dx=\"0\" dy=\"0\"/><feMerge><feMergeNode in=\"offsetBlur\"/><feMergeNode
```
Comment (zhimadaren) [inline community templates/Sensitive Data Discovery with Delphix/manifest.json:30]: ditto

**Pull request merged**